src/queue.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+"""File implements a basic queue data structure."""
+from __future__ import unicode_literals
+import dll
+
+
+class Queue(object):
+    """Queue class implements a simple queue data structure."""
+
+    def __init__(self, iterable=None):
+        self.dll = dll.DoublyLinkedList(iterable)
+
+    def enqueue(self, data):
+        """Add a value to the head of the queue."""
+        return self.dll.push(data)
+
+    def dequeue(self):
+        """Remove a value from the tail of the queue."""
+        return self.dll.remove(self.dll.tail.data)
```

Comment:
remove() is not the appropriate DLL method to be calling on here. Remove is an expensive loop through the whole list.

Also, if you are saying to remove the tail's data in order to dequeue, what happens when your list has two of that same value in it? e.g.
`Queue([1,2,3,2])`
What would happen when you call dequeue on this instance?